nixos/modules/services/networking/firewall.nix

```diff
@@ -538,10 +538,10 @@ in
         };
         protocols = mkOption {
           type = types.nullOr (types.listOf (types.enum
-            ["tcp" "udp" "icmp" "udplite" "esp" "ah" "sctp" ]
+            [ "tcp" "udp" "icmp" "udplite" "esp" "ah" "sctp" ]
```

Comment:
I would still not limit to an enum but make it dynamic. For instance I use `gre` and `ospf`, which is not in this list.